src/gui/properties/propertieswidget.ui

```diff
@@ -607,12 +607,6 @@
               <string>Information</string>
              </property>
              <layout class="QGridLayout" name="gridLayout">
-              <property name="topMargin">
```

Comment:
This probably shouldn't be in this PR.

Reply:
> 
> 
> This probably shouldn't be in this PR.

This often happens when you edit it using Qt Creator. Apparently, there are differences in the behavior of different versions...